Bug#163

remybaranx opened the pull request:
I fixed the bug #163, except the point (4) about html content of a node. See discussion in issue #163.
I also modified the `getSelectedNode()` method to raise a `QTasteTestFailException` when no node is selected.

dergo commented:
For me `getSelectedNode()` should return `null` instead of raising a `QTasteTestFailException`when no node is selected.

remybaranx commented:
Yes, you're right, it's better :-)

I did the modification and I have updated the playback script to handle the 'None' return when no node is selected.

remybaranx commented:
I have added 2 methods : 
- `selectNodeRe()` where node path elements are regular expressions
- `clearNodeSelection()` to clear the node selection

I have updated the playback demo by clearing the node selection at the beginning of the test.
